README.md

```diff
@@ -191,3 +191,49 @@ Class used to describe the certificates that should be maintained.
 ##### Parameters (all optional)
 
 * `domains`: Specifies the list of domains to include as SAN (Subject Alternative Names).
+
+### Functions
+
+#### Function: `dehydrated::ssl_cert_file`
+
+Function used to provide the `ssl_cert_file` path.
```

Comment:
Kudos for adding some doc, can we add this as [puppet-string doc](https://puppet.com/docs/puppet/6.18/puppet_strings_style.html#functions) in the code to avoid cluttering the README?

Maybe adding a bit of code in ModuleSync to [ensure that the doc is up-to-date](https://github.com/xaque208/puppet-bacula/blob/master/Rakefile#L27lL31) ?

Reply:
Its another topic. So, sure… later.

Reply:
I just added in-line doc and missing data types.